packages/hoppscotch-common/src/services/banner.service.ts

```diff
@@ -49,12 +50,18 @@ export class BannerService extends Service {
   public showBanner(banner: BannerContent) {
     this.bannerID = this.bannerID + 1
     this.bannerList.value.push({ id: this.bannerID, content: banner })
+
+    this.currentBannerID.value =
+      getBannerWithHighestScore(this.bannerList.value)?.id ?? null
+
```

Comment:
shouldn't we set the `currentBannerID` when `content` in generated, also the `content` will be the currentBanner right ? can we use that instead of `currentBannerID` @JoelJacobStephen

Reply:
@nivedin We need the id to remove the current banner through the `removeBanner` function. This can happen when a different service creates a banner but the header component removes the banner when a dismiss event is invoked. We can remove the current banner form anywhere by just accessing the id and sending it to the function. For the setting of the id, I will provide an assertion since `bannerList` cannot be null as the setting of id is done after pushing a banner to the `bannerList`. Will make that change.